modules/govuk/manifests/apps/ckan/db.pp

```diff
@@ -37,4 +37,15 @@
     rds                     => $rds,
     extensions              => ['postgis'],
   }
+
+  govuk_postgresql::db { 'pycsw':
```

Comment:
I wonder if it's better to call this `ckan_pycsw_production` just so it's obvious from the outside what the database is for.

Reply:
Could do